src/styles/pages/store/collections.module.scss

```diff
@@ -0,0 +1,28 @@
+@use 'src/styles/vars.scss' as vars;
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  margin: 0 auto;
+  max-width: 81rem;
+
+  .header {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+  }
+
+  .collections {
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: repeat(auto-fill, minmax(20rem, 1fr));
+    @media screen and (max-width: vars.$breakpoint-sm) {
+      grid-template-columns: 1fr;
+    }
+  }
+
+  .card {
+    flex: none;
+  }
```

Comment:
It doesn't look like this is needed; in inspect element it shows as greyed out, meaning that it's not doing anything, so removing it should be fine

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/91f7711d-e345-44f0-960f-b40d151ea619)

If you do remove it, it'll probably make an error about `styles.card` not being defined. In that case, you can just remove the `className={styles.card}` line as well